homeassistant/components/home_connect/__init__.py

```diff
@@ -268,3 +270,29 @@ async def update_all_devices(hass: HomeAssistant, entry: ConfigEntry) -> None:
             await hass.async_add_executor_job(device.initialize)
     except HTTPError as err:
         _LOGGER.warning("Cannot update devices: %s", err.response.status_code)
+
+
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    if config_entry.minor_version == 1:
```

Comment:
Let's also check for the normal version to be sure

Reply:
Do you mean I should check minor version `2` also?
It would be okey if I use `config_entry.minor_version <= 2`?

Reply:
No I mean, we should also check `version == 1`

Reply:
Oh, I understand, ty!